README.md

```diff
@@ -2,11 +2,9 @@
 
 > A modern web interface for OwnTracks made with Vue.js
 
-![Docker Pulls](https://img.shields.io/docker/pulls/owntracks/frontend)
+[![Docker Pulls](https://img.shields.io/docker/pulls/owntracks/frontend)](https://hub.docker.com/r/owntracks/frontend)
 
-<p style="text-align: center;">
-  <img src="docs/images/owntracks-ui.png" alt="OwnTracks UI">
-</p>
+![OwnTracks UI](docs/images/owntracks-ui.png)
```

Comment:
I was deliberately using a bit of HTML here for `text-align: center;` :wink:

Reply:
Sorry I rolled it back, I hoped it would fix the image not displaying on docker hub but it didn't 😞

Reply:
Fair point. That's most likely caused by the relative path `docs/images/owntracks-ui.png` which GitHub handles correctly but Docker Hub probably doesn't.

Reply:
Let's see if https://github.com/owntracks/frontend/commit/ed2118339da6a88daeb1eb6c4be5f8efeb5485d0 will fix this.

Reply:
Hooray!

![image](https://user-images.githubusercontent.com/19366641/65550706-153c6700-df18-11e9-8086-005b46f83ce4.png)